Microsoft.Bot.Builder.LUIS.Tests/LuisRecognizerTests.cs

```diff
@@ -0,0 +1,170 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Cognitive.LUIS;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Bot.Builder.LUIS.Tests
```

Comment:
I'm not clear how these tests could be run on the Build Machine. 

Please look at how we're doing this in the [other tests](https://github.com/Microsoft/botbuilder-dotnet/blob/master/tests/Microsoft.Bot.Builder.Ai.Tests/QnAMakerTests.cs#L15). And the solution [we're using](https://github.com/Microsoft/botbuilder-dotnet/blob/d28d218f78a735f6b43399844b04b3bc654fda23/tests/Microsoft.Bot.Builder.Tests/TestUtilities.cs#L40).

Reply:
Done